### PR TITLE
docs(atomic): rating facet edits

### PR DESCRIPTION
**Pull request by dlutzCoveo**

### CODE REVIEW

File: packages/atomic/src/components/facets/atomic-rating-facet/atomic-rating-facet.tsx [hunk 1]
```diff
@@ -109,6 +109,13 @@ export class AtomicRatingFacet
    * - Use a value that starts with `http://`, `https://`, `./`, or `../`, to fetch and display an icon from a given location.
    * - Use a value that starts with `assets://`, to display an icon from the Atomic package.
    * - Use a stringified SVG to display it directly.
+   *
+   * When using a custom icon, at least part of your icon should have the colour set to `fill="currentColor"`.
```
Comment (srosenberg-coveo) [inline packages/atomic/src/components/facets/atomic-rating-facet/atomic-rating-facet.tsx:113]:
```suggestion
   * When using a custom icon, at least part of your icon should have the color set to `fill="currentColor"`.
```

File: packages/atomic/src/components/facets/atomic-rating-range-facet/atomic-rating-range-facet.tsx [hunk 2]
```diff
@@ -101,6 +101,12 @@ export class AtomicRatingRangeFacet
    * - Use a value that starts with `http://`, `https://`, `./`, or `../`, to fetch and display an icon from a given location.
    * - Use a value that starts with `assets://`, to display an icon from the Atomic package.
    * - Use a stringified SVG to display it directly.
+   *
+   * When using a custom icon, at least part of your icon should have the colour set to `fill="currentColor"`.
```
Comment (srosenberg-coveo) [inline packages/atomic/src/components/facets/atomic-rating-range-facet/atomic-rating-range-facet.tsx:105]:
```suggestion
   * When using a custom icon, at least part of your icon should have the color set to `fill="currentColor"`.
```

**Review by srosenberg-coveo — APPROVED** [review]

File: packages/atomic/src/components/facets/atomic-rating-facet/atomic-rating-facet.tsx [hunk 3]
```diff
@@ -109,6 +109,13 @@ export class AtomicRatingFacet
    * - Use a value that starts with `http://`, `https://`, `./`, or `../`, to fetch and display an icon from a given location.
    * - Use a value that starts with `assets://`, to display an icon from the Atomic package.
    * - Use a stringified SVG to display it directly.
+   *
+   * When using a custom icon, at least part of your icon should have the color set to `fill="currentColor"`.
+   * This part of the SVG will take on the colors set in the following variables:
```
Comment (ThibodeauJF) [inline packages/atomic/src/components/facets/atomic-rating-facet/atomic-rating-facet.tsx:114]: I would maybe make variables clickable with this link https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

File: packages/atomic/src/components/facets/atomic-rating-facet/atomic-rating-facet.tsx [hunk 4]
```diff
@@ -87,11 +87,11 @@ export class AtomicRatingFacet
    */
   @Prop() public field!: string;
   /**
-   * The number of intervals to split the index for this facet.
+   * The number of intervals to split the index into and display in the facet. This property will define the number of rating icons displayed if the `maxValueInIndex` isn't assigned a value.
    */
   @Prop() public numberOfIntervals = 5;
   /**
-   * The maximum value of the field. This value is also used as the number of icons to be displayed.
+   * The maximum value in the field's index and the number of rating icons to display in the facet. If not assigned a value, this property will default to the same value as `numberOfIntervals`.
```
Comment (btaillon) [inline packages/atomic/src/components/facets/atomic-rating-facet/atomic-rating-facet.tsx:94]: I like this description

**Pull request merged**